### DISCUSSION TITLE
issue with 'retext-keywords' plugin

### DISCUSSION THREAD

**Discussion by github-Hamid** [retextjs/retext#92] [General]
Hello, I have a problem with 'retext-keywords' plugin. the problem is I have a word like 'mg/kg' and when I use the 'retext-keywords' plugins it removes '/' character and anything before it and just consider 'kg' as a keyphrase. Is there any way that I can get the whole 'mg/kg' word as a keyphrase by 'retext-keywords'?
Thank you

**Comment by wooorm** [comment]
I don't think mg/kg is one word?

**Reply by github-Hamid** [reply]
it's a unit and it was just an example. I also have a word like 'A/MS-H275Y' which is a virus name in the text  and it removes 'A/' and just keep 'MS-H275Y'. My question is, is there any way that 'retext-keywords'  does not remove '/' and everything before it?

**Reply by wooorm** [reply]
No, because it classifies words. And then looks for words. It doesn’t know about virus names or other non-human-language codes.
If you want these cases to be classified as words, you need to write a plugin to do that.
Either by walking the AST, or by integrating with the parser:

For example, in English, there is an exception for `w/` being seen as one word:

https://github.com/wooorm/parse-english/blob/504b3b3340facacf39126ff571924ed08557cb49/index.js#L14-L18

https://github.com/wooorm/parse-english/blob/504b3b3340facacf39126ff571924ed08557cb49/index.js#L195-L214
